File: program76.c
//accept one number and accept another 1 number and find out frequency of that number in n numbers
//return smallest number from that n numbers
//interview for sure
//positive and negative
#include<stdio.h>
#include<stdlib.h>

int Frequency(int Arr[],int iLength,int iNo)
{
    int iFre=0,iCnt=0;
    for(iCnt=0;iCnt<iLength;iCnt++)
    {
       if(Arr[iCnt]==iNo)
       {
         iFre++;
       }

    }
    return iFre;

}

int main()
{
    int *ptr=NULL;
    int iCnt=0,iRet=0;
    int iSize=0,iNo=0;

    printf("ENter number of elements\n");
    scanf("%d",&iSize);

    ptr=(int*)malloc(iSize*sizeof(int));
    printf("Enter the values ");

    for(iCnt=0;iCnt<iSize;iCnt++)
    {
        scanf("%d",&ptr[iCnt]);
    }

    printf("enter the element to calculate frequency\n");
    scanf("%d",&iNo);

    iRet=Frequency(ptr,iSize,iNo);

    printf("Frequency of  %d is %d\n",iNo,iRet);
    free(ptr);

    return 0;
}

File: program76.java
//find out the frequency of that number
//accept n numbers and perform addition of that numbers
import java.lang.*;
import java.util.*;

class ArrayR
{
    public int Frequency(int array[],int no)
    {
        int sum=0,fre=0;
        for(int i=0;i<array.length;i++)
        {
            if(array[i]==no)
            {
                fre++;
            }
            
        }
        return fre;
    }
}

class program76
{
    public static void main(String arg[])
    {
        int ret=0;
        Scanner sobj=new Scanner(System.in);
        System.out.println("Enter the size of the Array");
        int isize=sobj.nextInt();

        int []MyArray=new int[isize];

        System.out.println("Enter the elements of the Array");
        for(int i=0;i<isize;i++)
        {
            MyArray[i]=sobj.nextInt();
        }

        System.out.println("ENter number to calculate frequency");
        int no=sobj.nextInt();

        ArrayR obj=new ArrayR();
        System.out.println("Frequency of the number is:"+obj.Frequency(MyArray,no));
    }
}